index.php

```diff
@@ -42,6 +42,13 @@
     ]
 ];
 
+function priceModify(int $price) {
```

Comment:
```suggestion
function priceModify(int $price): string {
```
Она же стринг вернёт? Не забывай про типизацию.